### DISCUSSION TITLE
Schmid factor analysis for amphibole

### DISCUSSION THREAD

**Discussion by rockdef** [mtex-toolbox/mtex#1865] [Ask Anything] [answered]
Hi all, 

I want some help regarding the Schmid factor analysis.
I can get all the plots regarding the Schmid factor analysis from the MTEX website, but there are a few additional things I want to incorporate.
 
1. How can we construct stress (sigma 1) direction (if you don't know the compression axis, generally from naturally deformed samples)?
 
2. How can we get alpha and beta angle (deviation angles of loading directions from pole to foliation and lineation, respectively) values?
 
3. How to construct the IPF of the Schmid factor with contour lines including the value?

I would be happy if anyone wants to share code related to the 'Orthorhombic group', not the quartz. Thank you very much in advance!

**Comment by kilir** [comment, marked as answer]
Hi Bhupesh,

wrt 1) since in many cases in nature, deformation results from non-coaxial flow, knowing the instantaneous stretching axis ISA (which translates to prinicpal stresses for infinitesimally samll deformation increments) is often not straight forward, often impossible. If the strain path is not complicated, you can try to make a kinematic analysis to figure out vorticity, flow plane/apohyses from which you can derive the ISA.

wrt 2) in mtex, you can compute the angle between two vectors using `angle()`.

wrt 3) this has been asked here several times, in short, simply provide the values as the second argumen, e.g. `plot(list_of_crystal_directions,list_of_data,'contourf')`.

Hope this helps.

Cheers,
Rüdiger

**Reply by rockdef** [reply]
Dear Rüdiger,

Thank you very much for the great insight into thinking about it in more detail.
I am going through this. I hope I will do that!
